src/preload.sh

```diff
@@ -207,7 +207,7 @@ function get_resin_os_version() {
 # Usage: expand_image <image_path>
 function expand_image() {
     # Size will be increased by 110% of container size
-    IMAGE_ADD_SPACE=$(($CONTAINER_SIZE / 100 + 300))
+    IMAGE_ADD_SPACE=$(($CONTAINER_SIZE / 100 + $CONTAINER_SIZE))
```

Comment:
110% would be `$CONTAINER_SIZE / 10 + $CONTAINER_SIZE` or `$CONTAINER_SIZE * 1.1`